cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraScheduler.java

```diff
@@ -88,6 +93,9 @@ public void resourceOffers(final SchedulerDriver driver, final List<Offer> offer
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("> resourceOffers(driver : {}, offers : {})", driver, protoToString(offers));
         }
+
+        configuration.getSeedManager().syncSeeds(clock.now().getMillis());
```

Comment:
Please see the timing of syncSeeds. Where are 2 cases: 1. when fetching is skipped because delay is not passed. 2. when seeds are actually fetched. Timing is following:
```
cassandraCluster.getSeedManager().syncSeeds() took 265 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 25 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 1 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
cassandraCluster.getSeedManager().syncSeeds() took 0 ms
```
Case 1. doesn't take a lot of time in any case. Case 2. could potentially take several seconds to fetch is the endpoint is slow. Even if it will go to timeout, it should not take more than a minute for each dc. Hm.. It could potentially block the scheduler. Need to think ..